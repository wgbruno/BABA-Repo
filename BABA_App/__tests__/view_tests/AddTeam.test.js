import {expect, jest, test} from '@jest/globals';
import renderer from 'react-test-renderer';
import RegisterTeam from '../../View/AddTeam';

test("Register New Team Screen", function (){
    const at_tree = renderer.create(<RegisterTeam/>).toJSON();
    expect(at_tree).toMatchSnapshot();
})