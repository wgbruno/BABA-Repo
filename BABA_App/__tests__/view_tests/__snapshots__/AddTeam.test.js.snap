// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register New Team Screen 1`] = `
Array [
  <View
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#e8e8e8",
        "borderRadius": 5,
        "borderWidth": 1,
        "marginVertical": 5,
        "paddingHorizontal": 10,
        "width": "100%",
      }
    }
  >
    <TextInput
      onChangeText={[Function]}
      placeholder="Team Name"
      style={Object {}}
      value=""
    />
  </View>,
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "alignItems": "center",
          "borderRadius": 5,
          "marginVertical": 5,
          "padding": 10,
          "width": "100%",
        },
        Object {
          "backgroundColor": "#DD4D44",
        },
        Object {},
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "white",
            "fontWeight": "bold",
          },
          undefined,
          Object {},
        ]
      }
    >
      Register
    </Text>
  </View>,
]
`;
